### PR TITLE
Rasterization

### PR DESCRIPTION
**Pull request by dbrandenstein**

### CODE REVIEW

**Comment by dbrandenstein** [comment]
[[GFBio] Rasterization für Punktdaten (Heatmap)](https://trello.com/c/ajTGM5DC/994-gfbio-rasterization-f%C3%BCr-punktdaten-heatmap)

File: src/operators/rasterization.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+| `source`  | `SingleVectorSource` |
+
+## Parameters
+### Grid Rasterization
+
+| Parameter           | Type                | Description                                                | Example Value            |
+|---------------------|---------------------|------------------------------------------------------------|--------------------------|
+| `spatialResolution` | `SpatialResolution` | The spatial resolution of the grid/size of the grid cells. | `{"x": 10.0, "y": 10.0}` |
+| `originCoordinate`  | `Coordinate2D`      | The origin coordinate to which the grid is aligned.        | `{"x": 0.0, "y": 0.0}`   |
+| `gridSizeMode`      | `String`            | The mode how the grid resolution is interpreted.           | `"fixed"`                |
```
Comment (ChristianBeilschmidt) [inline src/operators/rasterization.md:21]: maybe say

`rixed` or `relative` instead of `String`

File: src/operators/rasterization.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+The `Rasterization` operator expects exactly one _vector_ input.
+
+| Parameter | Type                 |
+|-----------|----------------------|
+| `source`  | `SingleVectorSource` |
+
+## Parameters
+### Grid Rasterization
+
+| Parameter           | Type                | Description                                                | Example Value            |
```
Comment (ChristianBeilschmidt) [inline src/operators/rasterization.md:17]: The `type` is missing here

File: src/operators/rasterization.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+The following describes the types used in the grid rasterization parameters.
+
+The parameters `spatialResolution` and `originCoordinate` consist of two fields `x` and `y` which describe a resolution/position in x/y direction.
+
+For `gridSizeMode` the two options `fixed` and `relative` are available.
+`Fixed` means the `spatialResolution` is interpreted as a constant grid cell size. 
+`Relative` means the `spatialResolution` is used as a multiplier for a query's spatial resolution, making the resulting grid size adaptive to the query resolution.
+
+### Density Rasterization
```
Comment (ChristianBeilschmidt) [inline src/operators/rasterization.md:33]: Maybe say something about the cutoff point

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: src/operators/rasterization.md (hunk 2 above)
Comment (dbrandenstein) [reply src/operators/rasterization.md]: https://github.com/geo-engine/docs/pull/35/commits/4341cd366bfe287effb41f6b74fa40a45ccbda6e

File: src/operators/rasterization.md (hunk 3 above)
Comment (dbrandenstein) [reply src/operators/rasterization.md:41]: https://github.com/geo-engine/docs/pull/35/commits/4fdd1f9cbb6408fbcb8777da60f5059bd12fc97f

File: src/operators/rasterization.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+          "x": 10,
+          "y": 10
+        },
+        "gridSizeMode": "fixed",
+        "originCoordinate": {
+          "x": 0,
+          "y": 0
+        }
+      }
+    },
```
Comment (ChristianBeilschmidt) [inline src/operators/rasterization.md:76]: cf. comment in backend:

```suggestion
    "params": {
      "type": "grid",
      "spatialResolution": {
        "x": 10,
        "y": 10
      },
      "gridSizeMode": "fixed",
      "originCoordinate": {
        "x": 0,
        "y": 0
      }
    },
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: src/operators/rasterization.md (hunk 4 above)
Comment (dbrandenstein) [reply src/operators/rasterization.md]: https://github.com/geo-engine/docs/pull/35/commits/bcb6bf0ffe762e31a818ba9fa865e07602260df6

**Pull request merged**